docs/userguide/editing.md

```diff
@@ -27,73 +27,153 @@ rather than Google Sites.
+fork repository must be [configured to serve GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)
+from the branch and the Markdown pages root directory (`/docs`).  Note: there is a
+latency of several minutes before GitHub Pages renders new changes, and 
+[errors](https://docs.github.com/en/github/working-with-github-pages/about-jekyll-build-errors-for-github-pages-sites)
+prevent the changes from being rendered.
 
-See the sitemap linked from the bottom of the navigation bar.
+## Jekyll Theme Configuration
 
-Most pages have an automatic Table of contents. On a new page, after entering
-some contents, return to the very top of the page contents, select Insert/Table
-of contents, save, then change it to Right-aligned and turn on Wrap.
+Jekyll site-wide configurations are stored
+in the `_config.yml` file. The current theme being used is
+[Just the Docs](https://pmarsceill.github.io/just-the-docs/).
 
-## Common Styles
+Due to incompatibilities between the GitHub's built-in implementation of the
+CommonMark Markdown parser, the default Jekyll Markdown parser, Kramdown, is
+used.
 
-We want to use common styles for code samples, notes and such. Since Google
-Sites does not offer a site-wide CSS style sheet, please copy special items from
-here, paste and modify their text, rather than creating them from scratch.
+### Document Structure
 
-For headings, and for standard text styles like **bold**, *italic*,
-~~strike-through~~, ... please use standard headings styles from Sites.
+Major chapters have Introduction pages, and further sections in a chapter are
+subpages of that main chapter page. The navigation bar is generated from the
+information provided in the headers section of the Markdown file according to 
+the theme's [navigation annotations](https://pmarsceill.github.io/just-the-docs/docs/navigation-structure/).
 
-### Code
+### Syntax
 
-**New:** Use the Format menu styles for Code (inline) and Blockquote Code
-(multi-line).
+[Standard markdown](https://commonmark.org/) (CommonMark) should be preferred over custom
+HTML to create content wherever possible.
 
-**Obsolete:**
+### Common Styles
 
-For inline class/type/function/constant names and similar use Sites' Courier New
-font which is close enough to the Courier font we used to use.
+Styles are provided by the theme's settings, which can be modified in `_config.yml`.
 
-For a block of code, please copy/paste the following and edit its contents:
+### Code
 
-U16_NEXT(s, i, length, c)
-U16_PREV(s, start, i, c)
-U16_APPEND(s, i, length, c, isError)
+Use the markdown notation of [single backtick](https://spec.commonmark.org/0.29/#code-spans)
+code spans (inline) and [triple backtick](https://spec.commonmark.org/0.29/#fenced-code-blocks)
+fenced code blocks (multi-line).  
 
-### Notes
+#### Notes
 
 *Endianness is not an issue on this level because the interpretation of an
```

Comment:
In other words, this was/is not something noteworthy about code in the user guide, but it's an example of a "Note: blah..."

Reply:
Done.